backend/packages/Upgrade/src/api/services/ExperimentService.ts

```diff
@@ -283,6 +305,7 @@ export class ExperimentService {
     }
     return getConnection().transaction(async (transactionalEntityManager) => {
       const experiment = await this.findOne(experimentId, logger);
+      this.cacheService.delCache(CACHE_PREFIX.EXPERIMENT_KEY_PREFIX + experiment.context[0]);
```

Comment:
You should await as this returns a promise

Reply:
code updated